app/containers/FileTree.js

```diff
@@ -35,7 +35,7 @@ function buildTree(folderName, files, handleClick, forceOpen = false) {
         return <File key={value} name={filename} path={value} handleClick={handleClick} />;
       }
       return <UnsupportedFile key={value} name={filename} />;
-    }).valueSeq();
+    }).valueSeq().toArray();
```

Comment:
It's not "necessary" really, but to make the prop types be an array of elements it is.